base/expr.jl

```diff
@@ -366,9 +366,17 @@ function remove_linenums!(ex::Expr)
             isa(x, LineNumberNode) && return false
             return true
         end
+    elseif ex.head === :macrocall
+        # Replace line information embedded into macro calls with `nothing`
+        # Removing the argument entirely invalidates the expression
+        map!(ex.args, ex.args) do arg
+            arg isa LineNumberNode ? LineNumberNode(1, :none) : remove_linenums!(arg)
```

Comment:
These seems controversial, since these are values, not line numbers it’s removing. We could strip the one from the first argument (replacing `__source__`  with `nothing`), but I don’t think we should modify the other argument literals in the list.

>